### PR TITLE
INSTALL: add cd command; fix typo

### PR DESCRIPTION
**Pull request by larsnaesbye**
User needs to switch to Celestia directory before git submodule command.
Also a typo fix.

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -53,17 +53,18 @@ To build from sources please follow instructions below.
...
-* git clone https://github.com/CelestiaProject/Celestia
-* git submodule update --init
+1. git clone https://github.com/CelestiaProject/Celestia
+2. cd Celestia
+3. git submodule update --init
 
 If you have fmtlib and Eigen3 installed from other sources (OS repository or
 vcpkg) or don't want to build Celestia with SPICE support then you can skip the
-2nd step.
+2nd and 3rd steps.
```
Comment (375gnu) [inline INSTALL]: just `3rd step`.

**Comment by larsnaesbye** [comment]
@375gnu I'll remember in the future, OK? :)

**Comment by 375gnu** [comment]
> @375gnu I'll remember in the future, OK? :)

it's just
```
git rebase -i HEAD~2
replace the 2nd pick with f
git push -f
```

**Comment by larsnaesbye** [comment]
Done :)

**Pull request merged**